Stratis.Bitcoin/FullNode.cs

```diff
@@ -133,6 +143,13 @@ void RunLoop()
 						//TODO: 
 						Logs.FullNode.LogError("Block rejected: " + block.Error.Message);
 					}
+
+					if (!reorg && block.Error == null)
```

Comment:
hmm but there is no await in the consensus loop so the task is just fired and the loop continues.
It may not be the best behavior (which is why we need an eventloop that runs on a separate thread)
Also on the repo there is a flag updateifnew (from browsing the dbreez code seems like it wont update if the block is already stored)